aws-xray-recorder-sdk-spring/src/main/java/com/amazonaws/xray/spring/aop/AbstractXRayInterceptor.java

```diff
@@ -0,0 +1,102 @@
+package com.amazonaws.xray.spring.aop;
+
+import com.amazonaws.xray.AWSXRay;
+import com.amazonaws.xray.entities.Subsegment;
+import com.amazonaws.xray.exceptions.SegmentNotFoundException;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Pointcut;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
As an SDK convention we have been relying on `org.apache.commons.logging.Log` for our log framework, only because it is bundled with the AWS SDK already, and keeps the logging consistent between the two. I'd be open to switching to `slf4j` across the SDK if a good case could be made for diverging from the AWS SDK, though.